src/Multitenancy.Tracker/TopicSubscriptionTracker.cs

```diff
@@ -0,0 +1,77 @@
+﻿using Cassandra;
+using PushNotifications.Contracts;
+using System.Collections.Generic;
+
+namespace Multitenancy.Tracker
+{
+    public partial class TopicSubscriptionTracker : ITopicSubscriptionTracker
+    {
+        const string CreateTableTemplate = @"CREATE TABLE IF NOT EXISTS ""pushnot_subscriptions"" (cv counter, name varchar, PRIMARY KEY (name));";
+        const string IncrementTemplate = @"UPDATE ""pushnot_subscriptions"" SET cv = cv + 1 WHERE name=?;";
+        const string DecrementTemplate = @"UPDATE ""pushnot_subscriptions"" SET cv = cv - 1 WHERE name=?;";
+        const string GetTemplate = @"SELECT * FROM pushnot_subscriptions WHERE name=?;";
+
+        private readonly ISession _session;
+        private readonly PreparedStatement incrementTemplate;
+        private readonly PreparedStatement decrementTemplate;
+        private readonly PreparedStatement getTemplate;
+
+        public TopicSubscriptionTracker(ISession session)
+        {
+            _session = session;
+
+            _session.Execute(CreateTableTemplate);
+
+            incrementTemplate = session.Prepare(IncrementTemplate);
+            decrementTemplate = session.Prepare(DecrementTemplate);
+            getTemplate = session.Prepare(GetTemplate);
+        }
+
+        public void Increment(string name)
```

Comment:
what happens when `name` is null or empty?

Reply:
Added null checks for this